### ISSUE TITLE
Where can I modify batch size and learning rate?

### ISSUE THREAD

**Issue by Jackson8787** [NUSTM/VLP-MABSA#22]
I ran out the vram during the pre-training task .Where can I modify batch size and learning rate？
![230147663-33dde6f4-afa2-420e-9bae-c5cb23846658](https://user-images.githubusercontent.com/85683870/231237270-30923998-8026-41d9-8f1b-d636a81c865e.png)

**Comment by lyhuohuo** [comment]
You can add "--lr" and "--batch_size" parameter in MVSA_pretrain.sh or just modify line 312 and line 397 in pretrain.py.
